rats-tls/src/attesters/internal/etls_enclave_attester_load_all.c

```diff
@@ -43,13 +43,15 @@ enclave_tls_err_t etls_enclave_attester_load_all(void)
 	while ((etls_readdir(dir, &ptr)) != 1) {
 		if (!strcmp(ptr->d_name, ".") || !strcmp(ptr->d_name, ".."))
 			continue;
+		if (strncmp(ptr->d_name+strlen(ptr->d_name)-2, "so", 2))
+			continue;
 
 #ifdef OCCLUM
 		/* Occlum can't identify the d_type of the file, always return DT_UNKNOWN */
 		if (strncmp(ptr->d_name + strlen(ptr->d_name) - strlen(PATTERN_SUFFIX),
 			    PATTERN_SUFFIX, strlen(PATTERN_SUFFIX)) == 0) {
 #else
-		if (ptr->d_type == DT_REG) {
+		if (ptr->d_type == DT_REG || ptr->d_type == DT_LNK) {
```

Comment:
Why checking symbol link type?

Reply:
In libs dir, there are soft link of  instance libs which can be quoted by application.